File: 0two-sum/0two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        for i in range(len(nums)):
            new_arr = nums[i+1:]
            if (target - nums[i]) in new_arr:
                return [i, new_arr.index(target - nums[i])+1+i]

File: 0two-sum/0two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let i = 0;
    while(i < nums.length ){
        for (let j = i+1; j < nums.length; j++)
        {
            if (nums[i]+nums[j]==target)  
            {
                return [i,j];
            }
        }
        i++;
    }
    
};